src/components/HomePage.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Box, Button, Image, Link, Stack, Text } from "@chakra-ui/react";
+
+const HomePage = () => {
+    return (
+        <div>
+          <Box bg='#CBE7F3' position='absolute'>
+            <Image src='../assets/images/homePageLaptopImage.png' alt='Home Page Laptop Image'/>
+
+            <Text
+              position='absolute'
+              left='0%'
+              right='40.34%'
+              top='9.78%'
+              bottom='57.64%'
+              fontFamily='Inter'
+              fontStyle='normal'
+              fontWeight='400'
+              fontSize='60px'
+              fontHeight='73px'
+              display='flex'
+              alignItems='center'
+              textAlign='center'
+              >
+                Make School Easier<br/>With PiSpace
+            </Text>
+
+            <Text
+              position='absolute'
+              left='0%'
+              right='40.34%;'
+              top='39.51%'
+              bottom='22.3%'
+              fontFamily='Open Sans'
```

Comment:
Also, extending the theme and adding fonts to their respective elements might be better, though we can do it later. https://chakra-ui.com/community/recipes/using-fonts

Reply:
> Also, extending the theme and adding fonts to their respective elements might be better, though we can do it later. https://chakra-ui.com/community/recipes/using-fonts

Yes, I'll do that later